src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead tr');
+
+thead.addEventListener('click', event => {
+  const index = [...thead.children].indexOf(event.target);
+  const tbody = document.querySelectorAll('table tbody tr');
```

Comment:
it's a collection of rows so maybe you name it like `tbodyRows` or something

Reply:
fixed